mkdocs.yml

```diff
@@ -64,6 +64,7 @@ nav:
   - 'Setup': setup.md
   - 'Add Dependencies': add-dependencies.md
   - 'Update Dependencies': update-dependencies.md
+  - 'Gradle buildSrcVersions': gradle-buildsrcversions.md
```

Comment:
Shouldn't we name if `buildSrcLibs` here?

Reply:
I use buildSrcVersions here on purpose.
From a SEO perspective, we want people who already know buildSrcVersions too google exactly that and land on our page that explains by what it was replaced instead of my old medium articles describing buildSrcVersions